assets/eip-3525/LICENSE

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
+
+CC0 1.0 Universal
+
+    CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES NOT PROVIDE
+    LEGAL SERVICES. DISTRIBUTION OF THIS DOCUMENT DOES NOT CREATE AN
+    ATTORNEY-CLIENT RELATIONSHIP. CREATIVE COMMONS PROVIDES THIS
+    INFORMATION ON AN "AS-IS" BASIS. CREATIVE COMMONS MAKES NO WARRANTIES
+    REGARDING THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS
+    PROVIDED HEREUNDER, AND DISCLAIMS LIABILITY FOR DAMAGES RESULTING FROM
+    THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED
+    HEREUNDER.
```

Comment:
Everything in this repository is CC0, so this file can be left out.  This is especially true with the comment at the top of each of the .sol files.

Reply:
> This is especially true with the comment at the top of each of the .sol files.

Actually, it's required by the Solidity style guide! Please keep them in.

Reply:
This repository is under no compulsion to follow the Solidity style guide, especially when it is being dumb and requiring redundant files.  😀 

I would want to hear a far more compelling reason to keep this than, "because someone on the internet thinks it should be included in complete projects".